job-dsls/jobs/seed_jobs/kie_new_branches_seed_job.groovy

```diff
@@ -53,7 +53,6 @@ job("${folderPath}/a-seed-job-${baseBranch}") {
                     "job-dsls/jobs/**/compile_downstream_build.groovy\n" +
                     "job-dsls/jobs/**/prod_projects_downstream_production.groovy \n" +
                     "job-dsls/jobs/**/dailyBuild_jdk11_pipeline.groovy\n" +
-                    "job-dsls/jobs/**/dailyBuild_jdk11_prod_pipeline.groovy\n" +
```

Comment:
this removal is not related to community ;)

Reply:
sure - it is not. But this is no t used for ages. we don't do a daily build for prod any more for community.